package.json

```diff
@@ -12,12 +12,13 @@
     "postbuild:client": "mkdir -p dist/client/public && cp _redirects dist/client/_redirects && cp public/favicon.ico dist/client/favicon.ico && cp public/android-chrome-512x512.png dist/client/public/icon.png && cp -r .well-known dist/client/.well-known",
     "build:server": "cp -r src/lambda/ dist/server",
     "postbuild:server": "mkdir -p dist/server/server && mv dist/client/index.html dist/server/server/",
-    "lint": "eslint 'src/**/*.js' --quiet --fix",
+    "lint": "eslint . --quiet --fix",
     "clean": "rimraf ./dist ./.cache",
-    "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js",
+    "ci:lint": "eslint . -c ./.eslintrc.js",
```

Comment:
What's the point of setting the config file here? 🤔

Reply:
I don't think we need. It should take by default